src/shared/analytics/analytics.background.ts

```diff
@@ -323,6 +324,19 @@ function trackAppEvents({ account }: { account: Account }) {
     }
   });
 
+  emitter.on('formQuoteReceived', async ({ quote, formView, scope }) => {
+    const params = createParams({
+      request_name: 'swap_form_filled_out',
+      screen_name: scope === 'Swap' ? 'Swap' : 'Bridge',
+      client_scope: scope === 'Swap' ? 'Swap' : 'Bridge',
+      action_type: scope === 'Swap' ? 'Trade' : 'Send',
```

Comment:
Why is this different from `'transactionSent'` event? The whole point of introducing client_scope and action_type was that they're unrelated